todo_app/app.py

```diff
@@ -1,5 +1,6 @@
 from flask import Flask, render_template, request, redirect
-from todo_app.data.session_items import get_items, add_item, get_item, save_item, remove_item
+from todo_app.data.classes.Item import ItemStatus
+from todo_app.data.trello_items import get_items, add_item, update_item, remove_item
```

Comment:
Nice, the fact this change is so simple in this file suggests a really effective abstraction layer 👍